portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -22,35 +22,72 @@
 import java.util.ArrayList;
 import java.util.List;
 import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.sps.data.Comment;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   private final Gson gson = new Gson();
 
+  /**
+   * Gets the comments from datastore
```

Comment:
normally we don't add @param/@return documentation to @override functions, the reason being the javadoc should already have been specified where the function is originally declared and if we add it in here we're duplicating that information and there's a chance it could get out of sync later.

same for your other post/get functions.

Reply:
Done - Removed. Ahhh didn't know that but it makes sense. The process of writing the @param/@return did help me understand what exactly was going on in those functions better though :).